node/sync.go

```diff
@@ -83,37 +83,24 @@ func (d *Pegnetd) SyncBlock(ctx context.Context, height uint32) error {
 	// We should be able to rollback any one of these eblock syncs.
 	var err error
 	eblocks := make(map[string]*factom.EBlock)
-EntrySyncLoop: // Syncs all eblocks we care about and their entries
 	for k, v := range d.Tracking {
 		if eblock := dblock.EBlock(v); eblock != nil {
-			if err = eblock.Get(d.FactomClient); err != nil {
-				break
-			}
-			for i := range eblock.Entries {
-				if err = eblock.Entries[i].Get(d.FactomClient); err != nil {
-					break EntrySyncLoop
-				}
+			if err = eblock.GetEntries(d.FactomClient); err != nil {
+				return err
```

Comment:
TIL

Reply:
yeah the library is inconsistent. you can do eblock.GetEntries and it will get both the eblock and entries, but dblock.geteblocks doesn't work the same